vagrant-tools/run-tests.sh

```diff
@@ -0,0 +1,99 @@
+. ~/.bash_profile
+cd /vagrant/borg/borg
+. ../borg-env/bin/activate
+if which pyenv > /dev/null; then
+  # for testing, use the earliest point releases of the supported python versions:
+  pyenv global 3.4.0 3.5.0
+fi
+# otherwise: just use the system python
+
+testingPartitionSize=128M
+rootTestingDir="/var/tmp/borg-tests" # Also hardcoded in server configs in this directory
+mkdir -p "$rootTestingDir/rootfs"
+mountBase="/mnt/borg-tests"
+mkdir -p "$mountBase"
+
+rootFS="$(grep -oP '^[^ ]* / \K[^ ]*' /proc/mounts)"
```

Comment:
This works on Linux and should work on *BSD from what I can tell. If this fails, it fails with warnings but a blank list of filesystems (except "rootfs" which isn't a mount point at all, just a dir). OSX might not have these /proc and /lib endpoints, but OSX doesn't have stable FS drivers for these either.